config/locales/id.yml

```diff
@@ -0,0 +1,37 @@
+en:
```

Comment:
The `en` means English. Please rename your lang.

Reply:
eh sorry, forgot to update it

Reply:
updated. Thanks